### ISSUE TITLE
PINN software addition

### ISSUE THREAD

**Issue by dario-coscia** [idrl-lab/PINNpapers#11]
Hello👋🏻
Great job in putting a lot of PINN literature just in one place!

I saw you have a software section, and I suggest you look at [PINA](https://github.com/mathLab/PINA) if you want to add another one! PINA is an open-source Python library providing an intuitive interface for solving differential equations using PINNs, NOs or both together based on [PyTorch](https://pytorch.org/) and [PyTorchLightning](https://lightning.ai/docs/pytorch/stable/). In addition to implementing the state-of-the-art methodology, we provide advancement training strategies such as multiple device training, modern model compression techniques, gradient accumulation, and so on thanks to PyTorchLightning.

If you find the software interesting and helpful it would be amazing to enter the list! 😄🚀

Paper Link: https://joss.theoj.org/papers/10.21105/joss.05352
GitHub: https://github.com/mathLab/PINA
